core/src/main/resources/openapi/overview.html

```
@@ -62,7 +62,10 @@
 <body>
 
 <%
-def hasRootServerPath = {server -> com.predic8.membrane.core.openapi.util.UriUtil.getPathFromURL(uriFactory, server.url) == '' || com.predic8.membrane.core.openapi.util.UriUtil.getPathFromURL(uriFactory, server.url) == '/' }
+def effectivePath = { api, server ->
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)